tools/npm_version_check.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+const pkg = require('./package.json');
+const pkgVersion = pkg.version;
+const rules_nodejsVersion = process.env['BUILD_BAZEL_RULES_NODEJS_VERSION'];
+
+const getMajor = versionString => versionString.split('.')[0];
+
+// special case the version 0.0.0 so that we don't have to stamp builds
+// for dev and testing
+if (pkgVersion !== '0.0.0' && getMajor(pkgVersion) !== getMajor(rules_nodejsVersion)) {
+  throw new Error(`Expected package major version to equal @build_bazel_rules_nodejs major version
```

Comment:
It would be better to point to a wiki page and/or print some advice about how to remedy the problem. this feels like an edge that novice users can easily trip over and they won't know what build_bazel_rules_nodejs is.